owl/EASE.owl

```diff
@@ -1144,6 +1189,42 @@ The second issue relates to what is &quot;outside&quot;. The state is a configur
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE.owl#3DPosition -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#3DPosition">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#EuclideanPoint"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE.owl#hasTranslation"/>
+                <owl:someValuesFrom rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
```

Comment:
Perhaps this should be an exactly 1 restriction, rather than some, because a 3D position will have exactly one translation. There is of course a question of which reference frame, and which unit of measurement/scale, this translation is expressed in. However, hasTranslation is a data property to a string, and it's not explained that this string should also contain reference frame and scale.

Alternatively, several translations for a point may indicate some probabilistic approach, where a point is described by a collection of "particles", where each particle mass corresponds to probability. Again though, this isn't explained.

I'm giving the examples as a suggestion that there are cases where it is useful to store more translations for the same point, but we should explain if we have such a situation in mind, and if not, I think we can use the exactly 1 restriction.